### PR TITLE
dockerbuild: add Ubuntu Trusty support

### PR DESCRIPTION
**Pull request by szemere**
Added Ubuntu Trusty to the group of available images in dbld.
Based on the already implemented dbldb/images/xenial, minor changes:

[dbld/images/trusty/dev-dependencies.txt](dbld/images/trusty/dev-dependencies.txt)
-libsystemd-dev
+libsystemd-daemon-dev
-openjdk-8-jdk
+openjdk-7-jdk
-python-pep8

[dbld/images/trusty/Dockerfile](dbld/images/trusty/Dockerfile)
obs repo versions

[dbldb/images/trusty/README.md](dbldb/images/trusty/README.md)
xenial->trusty

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: dbld/images/trusty/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Assume that we have cloned syslog-ng's source into the `$HOME/syslog-ng`
+directory.  The following commands starts a container mounted with the
+source:
+
+```bash
+$ dbld/rules shell-trusty
+```
+
+You can also build an RPM using:
+
```
Comment (bazsi) [inline dbld/images/trusty/README.md]: deb

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by mitzkia** [comment]
Lgtm

**Pull request merged**